keycloak/keycloak-admin-ui
# events: search user events form, chips for searched fields and tests

## Motivation
Add Search for user events as per https://marvelapp.com/prototype/7ca9dhf/screen/73035918/handoff. 

## Brief Description
All functionality for searching user events is implemented as per designs above.

## Verification Steps
1. Go to /event configs page and enable events, e.g login, and logout.
2. Go to /events page and open search form, add search criteria e.g set event type to "LOGIN" and "LOGOUT and select 'Search events' button".
5. Verify that correct search results are displaying and the correct chips are displaying.
6.  Return to the search form and verify that the old search criteria persist. Remove event types and enter non-existing user id.
7. Verify the correct search results are displaying (no results should display/empty table) and that the correct chip for "User ID" is displaying. 
8. Select the 'refresh' button and verify that the search form is empty and all events display.
9. Repeat searching by user id and event types "LOGIN" and "LOGOUT".
10. Remove "User ID" category, verify that the removed user id is removed from the search form and that correct search results are displaying.
11. Remove "LOGIN" event type from "Event type" category, verify that the removed "LOGIN" is removed from the search form and that correct search results are displaying.

## Checklist:
